generators/client/templates/angular/src/main/webapp/app/admin/user-management/user-management-update.component.ts.ejs

```diff
@@ -98,6 +100,7 @@ export class UserMgmtUpdateComponent implements OnInit {
     }
 
     private updateUser(user: User): void {
+        user.id = this.editForm.get(['id']).value;
```

Comment:
is it necessary as we won't update the user ID ?

Reply:
not really. Will update later today